docs/version0.25.md

```diff
@@ -36,10 +37,46 @@ The following new features and notable changes since v 0.24.0 are included in th
 
 OpenJ9 release 0.25.0 supports OpenJDK 16. OpenJDK 16 with Eclipse OpenJ9 is *not* a long term support (LTS) release.
 
-Windows&reg; builds for Java&trade; 8 are now compiled with Microsoft&reg; Visual Studio 2013. The Visual Studio redistributable files included with the build are updated to match.
+Although it might be possible to build an OpenJDK 8 or OpenJDK 11 with OpenJ9 release 0.25.0, testing at the project is not complete and therefore support for new features that apply to these Java versions is not available.
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### ![Start of content that applies to Java 16](cr/java16.png) New JDK 16 features
+
+The following features are implemented in OpenJ9:
+
+- [JEP 338](https://openjdk.java.net/jeps/338): Vector API (incubator)
+    - The OpenJ9 project is extending this feature with optimizations.
+- [JEP 390](https://openjdk.java.net/jeps/390): Warnings for value-based classes
+- [JEP 395](https://openjdk.java.net/jeps/395): Records
+- [JEP 397](https://openjdk.java.net/jeps/397): Sealed Classes (second preview)
+
+The following features apply to OpenJ9 but are deferred to a future release:
+
+- [JEP 389](https://openjdk.java.net/jeps/389): Foreign linker API (incubator)
+- [JEP 393](https://openjdk.java.net/jeps/393): Foreign-memory access API (third incubator)
+
+The following features do not apply directly to OpenJ9 but are available in any builds of OpenJDK 16 with OpenJ9:
+
+- [JEP 347](https://openjdk.java.net/jeps/347): Enable C++ 14 language features
+- [JEP 380](https://openjdk.java.net/jeps/380): Unix-domain socket channels
+- [JEP 394](https://openjdk.java.net/jeps/394): Pattern matching for `instanceof`
+- [JEP 396](https://openjdk.java.net/jeps/396): Strongly encapsulate JDK internals by default
+- [JEP 392](https://openjdk.java.net/jeps/392): Packaging tool
+
+You can find the full list of features for JDK 16 at the [OpenJDK project](http://openjdk.java.net/projects/jdk/16/). Any remaining features that are listed by the OpenJDK project do not apply to OpenJ9.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Applications might be adversely affected by
+JEP 396 if they make use of internal APIs other than `sun.misc.Unsafe`. You should update your application to use standard APIs. To temporarily work around this problem, set `--illegal-access=permit` on the command line, which prints a warning that is similar to the following example when an illegal access call is made:
```

Comment:
It's more than just sun.misc.Unsafe. Classes in packages exported from modules are still accessible. Related to Unsafe there are the jdk.unsupported and jdk.unsupported.desktop modules, which are expected to be removed in some future version of Java.

jdk.unsupported exports:
- com.sun.nio.file (ExtendedCopyOption, ExtendedOpenOption, ExtendedWatchEventModifier, SensitivityWatchEventModifier)
- sun.misc (Signal, SignalHandler, Unsafe)
- sun.reflect (ReflectionFactory)

jdk.unsupported.desktop exports:
- jdk.swing.interop (6 classes)

Reply:
I'm not sure if we really want to expand all this here? Could we just say the following and leave users to read the spec changes themselves?

Applications might be adversely affected by JEP 396 if they make use of internal APIs ~~other than `sun.misc.Unsafe`~~. You should update your application to use standard APIs. To temporarily work around this problem, set `--illegal-access=permit` on the command line, which prints a warning that is similar to the following example when an illegal access call is made:

Reply:
Sounds good.